src/component/retrospect/template/list/CustomTemplateListItem.tsx

```diff
@@ -1,64 +1,139 @@
 import { css } from "@emotion/react";
 
-import { Button } from "@/component/common/button";
+import { BottomSheet } from "@/component/BottomSheet";
+import { Button, ButtonProvider } from "@/component/common/button";
 import { Card } from "@/component/common/Card";
+import { DropdownMenu } from "@/component/common/dropdownMenu/DropdownMenu";
+import { TextArea } from "@/component/common/input";
 import { Tag } from "@/component/common/tag";
 import { Typography } from "@/component/common/typography";
+import { useDeleteCustomTemplate } from "@/hooks/api/template/useDeleteCustomTemplate";
+import { usePatchTemplateTitle } from "@/hooks/api/template/usePatchTemplateTitle";
+import { useBottomSheet } from "@/hooks/useBottomSheet";
+import { useInput } from "@/hooks/useInput";
+import { useModal } from "@/hooks/useModal";
 
 type CustomTemplateListItem = {
+  id: number;
+  spaceId: number;
   title: string;
   tag: string;
   date: string;
   createRetrospect?: () => void;
 };
 
-export function CustomTemplateListItem({ title, tag, date, createRetrospect }: CustomTemplateListItem) {
+const MENU_EDIT = "edit-name";
+const MENU_DELETE = "delete";
+
+export function CustomTemplateListItem({ id, spaceId, title, tag, date, createRetrospect }: CustomTemplateListItem) {
+  const { open } = useModal();
+  const { openBottomSheet } = useBottomSheet();
+  const { value: templateTitle, handleInputChange: handleChangeTitle } = useInput(title);
+  const patchTemplateTitle = usePatchTemplateTitle(spaceId);
+  const deleteCustomTemplate = useDeleteCustomTemplate(spaceId);
```

Comment:
이런 식으로 구조분해할당으로 가져온 다음, 별칭을 지정해서 사용하면 변수에 `mutate`로 한번 더 접근을 안해도 좋을 것 같다는 생각이 들어요!

```suggestion
  const { mutate: deleteCustomTemplate } = useDeleteCustomTemplate(spaceId);
```

Reply:
넵 통일성을 위해 다른 부분들에서도 같이 수정해둘게요 !!